### PR TITLE
[DGPFUS-27] Develop an E-Commerce Store Database

### PR DESCRIPTION
**Pull request by Olabode969**

### CODE REVIEW

File: submissions/olabodeAmusan/Database/init.sql [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+name VARCHAR(250),
+amount DECIMAL(10,2) NOT NULL,
+PRIMARY KEY(id)
+);
+
+/*Table for Admins*/
+CREATE TABLE admins (
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(250),
+type INT NOT NULL,
```
Comment (Mayowa2020) [inline submissions/olabodeAmusan/Database/init.sql:31]: The admins' table will contain both the system admins of the food bag platform and the vendors. They both will be differentiated by type. The system admins can have a type of 0 and vendors a type of 1 when they are saved for differentiation. 

Boolean values are nothing but 0 and 1 i.e. either true or false. if the condition satisfies, it is true else it is false. So change line 3 to:  type BOOLEAN default TRUE.

File: submissions/olabodeAmusan/Database/init.sql [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+PRIMARY KEY(id)
+);
+
+/*Registered Customers*/
+CREATE TABLE customers (
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(250),
+phone_number VARCHAR(250) NOT NULL,
+address VARCHAR(250) NOT NULL,
+payment_method VARCHAR(250) NOT NULL,
```
Comment (Mayowa2020) [inline submissions/olabodeAmusan/Database/init.sql:44]: Change lines 44 to: payment_method ENUM('cash', 'pos', 'transfer, cheque') DEFAULT 'cash',

File: submissions/olabodeAmusan/Database/init.sql [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Card_number INT NOT NULL,
+card_name VARCHAR(250) NOT NULL,
+expiry_date DATE,
+cvc INT NOT NULL,
+PRIMARY KEY(id)
+);
```
Comment (Mayowa2020) [inline submissions/olabodeAmusan/Database/init.sql:50]: Remove lines 45 to 48

File: submissions/olabodeAmusan/Database/init.sql [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+/*Customers' order table*/
+CREATE TABLE orders (
+id INT NOT NULL AUTO_INCREMENT,
+vendor_id INT NOT NULL,
+customer_id INT NOT NULL,
+PRIMARY KEY(id)
+);
```
Comment (Mayowa2020) [inline submissions/olabodeAmusan/Database/init.sql:58]: The vendor_id and customer_id are foreign keys in this table, therefore reference ought to be made to their own tables here in this table.

**Review by Mayowa2020 — CHANGES_REQUESTED** [review]
Well done @Olabode969! Kindly make the requested changes and revert when done.

File: submissions/olabodeAmusan/Database/init.sql (hunk 4 above)
Comment (Olabode969) [reply submissions/olabodeAmusan/Database/init.sql:57]: Hello Ma,

I have made the corrections. Kindly review

**Review by Mayowa2020 — APPROVED** [review]
Well done @Olabode969! **Approved**

**Pull request merged**